.github/workflows/stellar-dominion-build-and-push.yml

```diff
@@ -12,11 +12,9 @@ on:
 
 jobs:
   extract-service-tag:
-    if: ${{ github.actor != 'github-actions[bot]' && github.event_name != 'push' || github.event.pusher.name != 'github-actions[bot]' }}
```

Comment:
Even on feature branch we want to:
* extract the current service tag
* build the docker image (for testing purposes for example)